images/linux/scripts/base/apt.sh

```diff
@@ -28,5 +25,8 @@ echo 'APT sources limited to the actual architectures'
 cat /etc/apt/sources.list
 
 apt-get update
-# Install aria2 , jq and apt-fast
-apt-get install aria2 jq apt-fast
+# Install aria2 , jq
+apt-get install aria2 jq
+
+# Install apt-fast using quick-install.sh
+bash -c "$(curl -sL https://git.io/vokNn)"
```

Comment:
nitpick: Maybe better to use the full url name instead of short preffix  - https://raw.githubusercontent.com/ilikenwf/apt-fast/master/quick-install.sh